src/data/equiv/mul_add.lean

```diff
@@ -482,30 +470,22 @@ variables [group_with_zero G]
 
 /-- Left multiplication by a nonzero element in a `group_with_zero` is a permutation of the
 underlying type. -/
+@[simps {fully_applied := ff}]
 protected def mul_left' (a : G) (ha : a ≠ 0) : perm G :=
 { to_fun := λ x, a * x,
   inv_fun := λ x, a⁻¹ * x,
   left_inv := λ x, by { dsimp, rw [← mul_assoc, inv_mul_cancel ha, one_mul] },
   right_inv := λ x, by { dsimp, rw [← mul_assoc, mul_inv_cancel ha, one_mul] } }
 
-@[simp] lemma coe_mul_left' (a : G) (ha : a ≠ 0) : ⇑(equiv.mul_left' a ha) = (*) a := rfl
-
-@[simp] lemma mul_left'_symm_apply (a : G) (ha : a ≠ 0) :
-  ((equiv.mul_left' a ha).symm : G → G) = (*) a⁻¹ := rfl
```

Comment:
Do the generated lemmas use `λ x, a * x` or `(*) a`? The latter seems more desirable, but I don't know about about `simps` to know if its what actually happens.